### ISSUE TITLE
Point of Sales

### ISSUE THREAD

**Issue by rmehta** [frappe-apps/pos#2]
Screen to pick items, add taxes and make a bill

**Comment by jcs98** [comment]
If possible, this screen can be synced with the other apps in this family.  
For example, once the bill is made and the payment is confirmed, the volume of the corresponding items sold can be updated in the inventory and the amount can be updated in the accounts.

**Comment by NirmalKumarJoshi** [comment]
@jcs98 This is obvious that after the payment the items and other records will be updated.

**Comment by NirmalKumarJoshi** [comment]
Yes right @rmehta, the first motive should be better user experience. The models and screens can be re-used with different backend but not making the application complex.

**Comment by NirmalKumarJoshi** [comment]
@rmehta We can also have a separate page for paid and overdue bills. This will keep track of the bills and makes it easy for the user to keep the record.
